### PR TITLE
Usercard : permit to choose the entity that publish (#2526)

### PR DESCRIPTION
**Pull request by vlo-rte**
Release notes : 

In Features section : 

#2526 : Usercard : permit to choose the entity that publish

### CODE REVIEW

File: ui/main/src/assets/i18n/en.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
     "endDateBeforeStartDate": "End date must be after start date",
     "featureNotConfigured": "Feature not configured",
     "confirmSettings": "CONFIRM SETTINGS",
+    "cardEmitter": "Card emitter",
```
Comment (freddidierRTE) [inline ui/main/src/assets/i18n/en.json:58]: EMITTER

File: ui/main/src/assets/i18n/fr.json [hunk 2]
```diff
@@ -55,6 +55,7 @@
     "endDateBeforeStartDate": "La date de fin doit être postérieure à la date de début",
     "featureNotConfigured": "Fonctionnalité non configurée",
     "confirmSettings": "ENREGISTRER LA CONFIGURATION",
+    "cardEmitter": "Emetteur de la carte",
```
Comment (freddidierRTE) [inline ui/main/src/assets/i18n/fr.json:58]: EMETTEUR

**Comment by freddidierRTE** [comment]
Add translation for nl ,  google translate says : EMITTER

File: ui/main/src/assets/i18n/en.json (hunk 1 above)
Comment (vlo-rte) [reply ui/main/src/assets/i18n/en.json]: Done.

**Comment by vlo-rte** [comment]
> Add translation for nl , google translate says : EMITTER

Done.

**Pull request merged**